mage_ai/orchestration/pipeline_scheduler.py

```diff
@@ -44,6 +44,40 @@
 MEMORY_USAGE_MAXIMUM = 0.95
 
 
+def stop_pipeline_run(
```

Comment:
You can put the method below PipelineScheduler class since other methods are below this class.